### PR TITLE
Adopter fosterer form answers

### PR DESCRIPTION
**Pull request by kasugaijin**
# 🔗 Issue
https://github.com/rubyforgood/homeward-tails/issues/1248
Adds ability for adtopers and fosterers to see their latest form submission's form answers so they can review it for accuracy. 
adopter-fosterer-form-answers

# ✍️ Description

# 📷 Screenshots/Demos

### CODE REVIEW

File: app/views/organizations/adopter_fosterer/form_answers/index.html.erb
Comment (kasugaijin) [inline app/views/organizations/adopter_fosterer/form_answers/index.html.erb:1]: Update the form answers to a partial that can be user here and in the staff review of form answers. Do this once base branch is merged.

File: config/routes.rb
Comment (kasugaijin) [inline config/routes.rb:1]: I re-arranged the routes file as it was getting ugly. Not directly related to this feature but sorely needed. I doubled checked to ensure I did not inadvertantly delete any routes. I did delete those that were no longer needed related to profile (the old adopter profile)

File: app/controllers/organizations/adopter_fosterer/form_answers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module Organizations
```
Comment (kasugaijin) [inline app/controllers/organizations/adopter_fosterer/form_answers_controller.rb:1]: I decided we only need to show the latest form answers (form submission) to adopters and fosterers. They just need to validate accuracy of the current 'snapshot' of data and don't need to see prior submissions.

**Review by jmilljr24 — APPROVED** [review]
@kasugaijin lgtm!

**Pull request merged**